docker-compose.yml

```diff
@@ -0,0 +1,22 @@
+version: "3.8"
+services:
+  oapen-engine :
+    build: ./oapen-engine/
+  db:
+    image: postgres:14.1-alpine
+    restart: always
+    environment:
+    # TODO change
+      - POSTGRES_USER=postgres
```

Comment:
I suggest changing this sooner rather than later. a simple COPY is all you need.